### DISCUSSION TITLE
Tailwind doesn't work in nested Lerna packages when the custom colors are added

### DISCUSSION THREAD

**Discussion by sviat-barbutsa** [tailwindlabs/tailwindcss#9764] [Help] [answered]
Hi, I'm using Lerna for the react monorepo architecture and I have 2 projects: A shared component library and the app that uses this library. Both use Tailwind
I configured tailwind to parse the content from the UI components library and it was working, however, when I added the custom colors into the _theme_ section (talwind.config.js) it stopped working. 
Could someone, please, give me some idea on how to solve /handle it, etc.?

I appreciate any help:

Here is the structure of the project
```
root
 - packages
 -- library
 -- app
```

Here is the tailwind configuration for **app**
_root/packages/app/tailwind.config.js_

```
module.exports = {
  relative: true,
  content: [
    './pages/**/*.{js,ts,jsx,tsx}', // app's content
    './components/**/*.{js,ts,jsx,tsx}',  // app's content
    '../library/src/**/*.{js,ts,jsx,tsx}', // here is the path to the library's content that contains tailwind and must be parsed as well
  ],
  darkMode: 'class',
  theme: {
    screens: {
      xs: '640px',
      sm: '768px',
      md: '1024px',
      lg: '1280px',
      xl: '1536px',
    },
    colors: { // if I add this - everything stops working
      primary: 'purple',
      secondary: 'orange',
    },
  },
  plugins: [],
};
```

Here is the postcss.config.js
_root/packages/app/postcss.config.js_
```
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': {},
    tailwindcss: {},
    autoprefixer: {},
    ...(process.env.NODE_ENV === 'production' ? { cssnano: {} } : {}),
  },
};
```

**Comment by wongjn** [comment, marked as answer]
Adding `colors` into `theme` [completely replaces the default colors that come with Tailwind](https://tailwindcss.com/docs/customizing-colors#using-custom-colors). You may be after [adding additional colors to the default palette](https://tailwindcss.com/docs/customizing-colors#adding-additional-colors) which you can do by defining your colors in `theme.extend.colors`:
```diff
       lg: '1280px',
       xl: '1536px',
     },
-    colors: { // if I add this - everything stops working
-      primary: 'purple',
-      secondary: 'orange',
-    },
+  },
+  extend: {
+    colors: {
+      primary: 'purple',
+      secondary: 'orange',
+    },
   },
   plugins: [],
 };
```

**Reply by sviat-barbutsa** [reply]
Thank you so much @wongjn now everything works like a charm.
You really helped me a lot. I spent the whole day struggling with different configurations etc. I was thinking that it was a Lerna problem. Didn't have even a small idea that I could have configured tailwind incorrectly. Missed that nuance in the docs, my bad
